### PR TITLE
alert-button completed

### PR DESCRIPTION
**Pull request by som749898**
completed alert and button component of sky ui component library

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.alert {
+  margin: 0.625rem;
+  padding: 0.938rem;
+  display: flex;
```
Comment (aniketsingh13) [inline Components/Alert/alert.css:4]: you can use utility for flex

**Review by aniketsingh13 — APPROVED** [review]
consider the changes other thing look good to me

**Review by hiteshverma27 — APPROVED** [review]
looks good

File: Components/Alert/alert.css (hunk 1 above)
Comment (som749898) [reply Components/Alert/alert.css:4]: I have used class alert as a utility in the whole HTML structure where it contains flex so I didn't include it as an extra utility in the HTML structure.

**Pull request merged**